packages/kbn-cli-dev-mode/src/get_server_watch_paths.ts

```diff
@@ -43,7 +43,7 @@ export function getServerWatchPaths({ pluginPaths, pluginScanDirs }: Options) {
         fromRoot('src/core'),
         fromRoot('src/legacy/server'),
         fromRoot('src/legacy/utils'),
-        fromRoot('config'),
+        // fromRoot('config'),
```

Comment:
Yeh, so, not sure how to best handle this. 

The dev server automatically restarts currently whenever the config changes which breaks interactive setup mode since that's exactly what we're doing.

Reply:
Assumint it's just for us, to test our interactive setup mode, do you think it'd be enough to use something like this?
```bash
$ yarn start --no-watch --interactiveSetup.forceSetup=true
```

If we need something more sophisticated then we may need to extend `Watcher` class to allow us to temporarily disable watching for a particular path.

Reply:
Ah nice, I'm happy with `--no-watch` for now to keep things simple